lib/customers.js

```diff
@@ -468,11 +468,11 @@ function getCustomersList () {
     c.sanctions_at, c.sanctions_override, t.tx_class, t.fiat, t.fiat_code, t.created, 
     row_number() over (partition by c.id order by t.created desc) as rn,
     sum(case when t.id is not null then 1 else 0 end) over (partition by c.id) as total_txs,
-    coalesce(sum(t.fiat) over (partition by c.id), 0) as total_spent
+    coalesce(sum(case when error_code is distinct from 'operatorCancel' then t.fiat else 0 end) over (partition by c.id), 0) as total_spent
```

Comment:
Why error_code distinct from 'operatorCancel'? Are all other errors still counting towards the total?